posthog/cdp/templates/__init__.py

```diff
@@ -23,38 +23,40 @@
 from .engage.template_engage import template as engage, TemplateEngageMigrator
 from .zendesk.template_zendesk import template as zendesk
 from .knock.template_knock import template as knock
+from .activecampaign.template_activecampaign import template as activecampaign
 from .google_cloud_storage.template_google_cloud_storage import (
     template as google_cloud_storage,
     TemplateGoogleCloudStorageMigrator,
 )
 
 
 HOG_FUNCTION_TEMPLATES = [
-    slack,
-    webhook,
-    hubspot,
+    activecampaign,
+    avo,
+    aws_kinesis,
+    braze,
+    clearbit,
     customerio,
+    engage,
+    gleap,
+    google_cloud_storage,
+    google_pubsub,
+    hubspot,
     intercom,
-    posthog,
-    sendgrid,
-    aws_kinesis,
-    zapier,
-    salesforce_create,
-    salesforce_update,
+    knock,
+    loops,
+    mailgun,
     mailjet_create_contact,
     mailjet_update_contact_list,
-    clearbit,
-    mailgun,
-    loops,
+    posthog,
     rudderstack,
-    avo,
-    gleap,
-    google_pubsub,
-    engage,
+    salesforce_create,
+    salesforce_update,
+    sendgrid,
+    slack,
+    webhook,
+    zapier,
     zendesk,
-    google_cloud_storage,
-    braze,
-    knock,
 ]
```

Comment:
sorted this alphabetically

Reply:
We might want an alternative way to order the list (via some actual usage data?). It's very messy now. I'd also suggest putting the name of the integration as the first word to make it easy to scan.

Reply:
I've reverted the change for now.